core/common/src/tile/IModelTileIO.ts

```diff
@@ -33,7 +33,7 @@ export enum CurrentImdlVersion {
    * front-end is not capable of reading the tile content. Otherwise, this front-end can read the tile content even if the header specifies a
    * greater minor version than CurrentVersion.Minor, although some data may be skipped.
    */
-  Major = 22,
+  Major = 23,

```

Comment:
Need to re-run extract-api for imodeljs-common.

Reply:
ran it again.